### ISSUE TITLE
What settings does the demo use?

### ISSUE THREAD

**Issue by occasionallydavid** [JaidedAI/EasyOCR#701]
The demo widget ( https://www.jaided.ai/easyocr/ ) produces much higher confidence scores than my runs of easyocr locally. Would it be possible to update the widget to show the parameters it uses? I'd like to understand the difference between my runs and its runs, and the source code for the demo does not seem to be on GitHub.

Thanks!

**Comment by Ktos93** [comment]
Yeah web demo works better than local install. I tried some preprocessing but its stil worse than web
